cpp/src/io/csv/csv_reader.cu

```diff
@@ -909,11 +909,12 @@ gdf_error uploadDataToDevice(const char* h_uncomp_data, size_t h_uncomp_size, ra
 	// Update the record starts to match the device data (skip missing records, fix offset)
 	for (gdf_size_type i = first_row; i <= first_row + raw_csv->num_records; ++i)
 		h_rec_starts[i] -= start_offset;
+	RMM_TRY(RMM_REALLOC(&raw_csv->recStart, sizeof(cu_recstart_t) * (raw_csv->num_records + 1), 0));
```

Comment:
`RMM_REALLOC` is currently broken as it deletes the existing data. See https://github.com/rapidsai/rmm/issues/5

A fix is in the works https://github.com/rapidsai/rmm/pull/8

CC @harrism

Reply:
@jrhemstad That's okay in this case, because I copy new data over in the next line.